src/vss_tools/vspec/vssexporters/vss2samm/helpers/samm_concepts.py

```diff
@@ -11,7 +11,7 @@
 
 from rdflib import URIRef
 
-from ..config import config as cfg
+from ..config import Config
 from . import string_helper as str_helper
 from .namespaces import samm_base_namespace, samm_output_namespace
```

Comment:
At this stage, samm_output_namespace will be in the form: `@prefix : <urn:samm:None:None#> .`

Expected should be like: `@prefix : <urn:samm:com.covesa.vss.spec:5..0.0#> .`